valhalla/baldr/traffictile.h

```diff
@@ -108,7 +112,12 @@ static_assert(sizeof(Speed) == sizeof(uint16_t),
 #ifndef C_ONLY_INTERFACE
 namespace {
 static constexpr volatile Speed INVALID_SPEED{0, 0, INVALID_SPEED_AGE_BUCKET, 0};
-}
+
+// Assert these constants are the same
+// (We want to avoid including this file in graphconstants.h)
+static_assert(MAX_TRAFFIC_SPEED_KPH == valhalla::baldr::kMaxTrafficSpeed,
```

Comment:
perhaps there was another reason why you didnt want to use the value from graphconstants? i was kind of thinking a static assert that made sure the value wasnt above `1<<8 - 1`

Reply:
See comment at https://github.com/valhalla/valhalla/pull/2325/files#r412394756

There is also the test at https://github.com/valhalla/valhalla/pull/2325/files#diff-7db274cd70eab71e43c6dc96bff4d5cdR69